### ISSUE TITLE
eaphammer hangs on 'Saving Current IP Configuration'

### ISSUE THREAD

**Issue by couldbejake** [s0lst1c3/eaphammer#135]
**Describe the bug**
After installing dependencies and running eaphammer, the script hangs on 'Saving Current IP Configuration' with no error messages. I've also tried running the above with a second wifi adapter identified by wlan1 with no success.

**To Reproduce**

1. ./kali-setup
2. ./eaphammer --cert-wizard
3. ./eaphammer -i wlan0 --channel 4 --auth wpa-eap --essid ECorp --creds

**Preview**

root@kali:~/eaphammer/eaphammer# ./eaphammer -i wlan1 --channel 4 --auth wpa-eap --essid CorpWifi --creds

[ ? ] Am I root?
[ * ] Checking for rootness...
[ * ] I AM ROOOOOOOOOOOOT
[ * ] Root privs confirmed! 8D
[ * ] Saving current iptables configuration...

**Command string(s) used**

./eaphammer -i wlan1 --channel 4 --auth wpa-eap --essid 

**EAPHammer verbose output (using the --debug flag)**

    "email": null,
    "essid": "CorpWifi",
    "fragm_threshold": null,
    "greenfield": false,
    "hostile_portal": false,
    "ht40": null,
    "ht_delayed_block_ack": false,
    "hw_mode": null,
    "interface": "wlan0",
    "interface_pool": null,
    "karma": false,
    "key_length": 2048,
    "known_beacons": false,
    "known_ssids": null,
    "known_ssids_file": null,
    "ldpc": false,
    "locale": null,
    "loud": false,
    "lsig_txop_prot": false,
    "mac_blacklist": null,
    "mac_whitelist": null,
    "manual_config": null,
    "max_num_stations": null,
    "max_spatial_streams": null,
    "negotiate": "balanced",
    "not_after": 94608000,
    "not_before": 0,
    "obss_interval": null,
    "org": null,
    "org_unit": null,
    "owe_transition_bssid": null,
    "owe_transition_ssid": null,
    "password": null,
    "peap_version": null,
    "pivot": false,
    "pmf": null,
    "pmkid": false,
    "private_key": null,
    "private_key_passwd": null,
    "psk_capture_file": null,
    "reap_creds": true,
    "remote_rig": null,
    "require_ht": false,
    "rts_threshold": null,
    "save_config": null,
    "save_config_only": null,
    "self_signed": false,
    "server_cert": null,
    "short_gi": false,
    "smps": null,
    "ssid_blacklist": null,
    "ssid_whitelist": null,
    "state": null,
    "troll_defender": false,
    "use_max_a_msdu_length": false,
    "user_list": null,
    "wmm": false,
    "wordlist": "/root/eaphammer/eaphammer/wordlists/rockyou.txt",
    "wpa_passphrase": null,
    "wpa_version": null
}
[ * ] Saving current iptables configuration...

**EAPHammer Version (./eaphammer --version)**

Paste output here.

**OS (the OS from which you are running EAPHammer)**

Kali Linux ARM for the Raspberry Pi 3: kali-linux-2019.1-rpi3-nexmon

**Configuration of Target Access Point (if known / applicable)**

ESSID: CorpWifi
Channel: 4
Authentication: WPA-EAP

**Comment by couldbejake** [comment]
It looks to be an issue with the command 'iptables-save'.

**Comment by s0lst1c3** [comment]
Hey @couldbejake, I noticed that you're using Kali Linux ARM. Since I'm not able to reproduce this on any of the supported operating systems listed in the docs, I'm going to close this issue for now. However, I have added a new issue requesting support for this OS, and I imagine this issue will be resolved once support is added. In the meantime, I suggest using Raspbian if you want to use the Raspberry Pi platform, since it is a supported platform and EAPHammer has been tested to work on it.
